dub.json

```diff
@@ -21,17 +25,33 @@
         },
     },
 
+    "subPackages": [
+        {
+            "name": "configure",
+            "targetType": "executable",
+            "targetPath": "bin",
+            "sourcePaths": [],
+            "sourceFiles": ["configure.d"]
+        }
+    ],
+
     "configurations": [
         {
             "name": "default",
             "mainSourceFile": "dstep/main.d",
             "targetName": "dstep",
             "sourcePaths": ["dstep", "clang"],
             "importPaths": ["dstep", "clang"],
+
+            "preBuildCommands-posix": [
+                "(! [ -s config.txt ] && ./configure) || true"
```

Comment:
The way it is currently used it makes more sense to call it something like `linker_flags.txt` as format is not reusable for any other kind of configuration.